201-web-app-loganalytics/azuredeploy.json

```diff
@@ -0,0 +1,180 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "skuName": {
+      "type": "string",
+      "metadata": {
+        "description": "Which Pricing tier our App Service Plan to"
+      },
+      "defaultValue": "S1"
+    },
+    "skuCapacity": {
+      "type": "int",
+      "metadata": {
+        "description": "How many instances of our app service will be scaled out to"
+      },
+      "defaultValue": 1
+
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "appName": {
+      "type": "string",
+      "defaultValue": "[uniqueString(resourceGroup().id)]"
+    }
+  },
+  "variables": {
+    "appServicePlanName": "[toLower(concat('asp-', parameters('appName')))]",
+    "webSiteName": "[toLower(concat('wapp-', parameters('appName')))]",
+    "appInsightName": "[toLower(concat('appi-',parameters('appName')))]",
+    "appInisghtsReference": "[concat('microsoft.insights/components/',variables('appInsightName'))]",
+    "logAnalyticsName": "[toLower(concat('la-',parameters('appName')))]"
+  },
+  "resources": [
+    {
```

Comment:
check the BP guide for sort order of properties, each resource seems to vary a bit:

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties